contrib/ros/worlds/__init__.py

```diff
@@ -0,0 +1,5 @@
+import os.path as osp
+
+import contrib.ros
+
+model_dir = osp.join(osp.dirname(contrib.ros.__file__), 'worlds/models')
```

Comment:
* Please never import system packages in `__init__.py`
* Please only import individual symbols in `__init__.py`, never whole packages.
* Defining symbols in `__init__.py` is a last resort. Why not define `MODEL_DIR` in the `World` class and then use it as `World.MODEL_DIR`?